Cargo.toml

```diff
@@ -101,11 +101,11 @@ fnv = "1.0.7"
 fs_extra = "1.3.0"
 futures = "0.3.26"
 glyph_brush = "0.7.7"
-hashbrown = { version = "0.13.2", features = ["bumpalo"] }
+hashbrown = { version = "0.14.3" }
```

Comment:
We were not using the bumpalo feature.

Reply:
I also thought at first that hashbrown 0.14.3 was no longer using ahash but I was wrong.